pkg/server/migration_test.go

```diff
@@ -363,7 +363,7 @@ func TestUpgradeHappensAfterMigrations(t *testing.T) {
 					for i := 0; i < N; i++ {
 						runtime.Gosched()
 					}
-					require.True(t, st.Version.ActiveVersion(ctx).Less(clusterversion.Latest.Version()))
+					require.True(t, st.Version.ActiveVersion(ctx).LessEq(clusterversion.Latest.Version()))
```

Comment:
@RaduBerinde could you confirm that this change is okay to do, i.e.
- current assertion is "require {active version} < {latest version}"
- but (I think?) assertion should be "require {active version} <= {latest version}"
thanks!

Reply:
okay, this passed for me as well. PR now keeps this test as-is.

(not sure why it failed before -- maybe I didn't update in all the right places, which caused the version to be off in one of the in-progress states?)

anyhow -- appreciate you spending the time digging into it (sorry for the churn)
<!-- Reviewable comment -OB8BtCx9ED6lSSWidE7:b-yetqea -->
<!-- Sent from Reviewable.io -->